src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -329,44 +369,72 @@ public void setSessionEnabled(boolean sessionEnabled) {
     }
 
     public Config build() {
-        Clients client = new Clients();
-        JaxRsConfig config = new JaxRsConfig();
+        // either the whole Config is built from a ConfigFactory class, or we initialize a default JaxRsConfig
+        final JaxRsConfig config;
+        if (configClass != null && !configProperties.isEmpty()) {
+            config = (JaxRsConfig) ConfigBuilder.build(configClass, configProperties);
```

Comment:
Are we sure the `config` generated will always be of this type? What if a user implements a ConfigFactory that returns a Config and not a JaxRsConfig?
Or we can simply expect user to do that, the error would be easy to correct if the user makes a mistake... just voicing some thoughts, but I think it can stay like that if  that's what you intended to do :)